install/helm/choreo-control-plane/Chart.lock

```diff
@@ -3,4 +3,4 @@ dependencies:
   repository: https://charts.jetstack.io
   version: v1.16.2
 digest: sha256:dd6953d54aeda0d8a27f5089a45eb6c7eff8de6a038c2c4a0128cec2129c53a1
-generated: "2025-05-09T09:36:17.45767+05:30"
+generated: "2025-05-26T14:39:59.596749+05:30"
```

Comment:
do we need to commit this chart.lock file? I remember last time we discussed, not to have it

Reply:
As discussed offline, lets keep this chart.lock file.